### PR TITLE
QDB-3317 Pinned columns

### PR DESCRIPTION
**Pull request by solatis**

### CODE REVIEW

File: src/main/c++/net/quasardb/qdb/jni/byte_buffer.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <iostream>
```
Comment (mkurdej) [inline src/main/c++/net/quasardb/qdb/jni/byte_buffer.cpp:1]: Why? Oh why are you killing innocent kittens?

File: src/test/java/net/quasardb/qdb/ts/WriterExtraTablesBenchmarkTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.*;
+
+import net.quasardb.common.TestUtils;
+import net.quasardb.qdb.ts.*;
+import net.quasardb.qdb.*;
+import net.quasardb.qdb.exception.InvalidIteratorException;
+import net.quasardb.qdb.exception.InvalidArgumentException;
+
+// @State(Scope.Benchmark)
```
Comment (mkurdej) [inline src/test/java/net/quasardb/qdb/ts/WriterExtraTablesBenchmarkTest.java]: Why is it all commented?

File: src/test/java/net/quasardb/qdb/ts/WriterAppendBenchmarkTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.*;
+
+import net.quasardb.common.TestUtils;
+import net.quasardb.qdb.ts.*;
+import net.quasardb.qdb.*;
+import net.quasardb.qdb.exception.InvalidIteratorException;
+import net.quasardb.qdb.exception.InvalidArgumentException;
+
+// @State(Scope.Benchmark)
```
Comment (mkurdej) [inline src/test/java/net/quasardb/qdb/ts/WriterAppendBenchmarkTest.java]: Again. Why is it all commented?

File: test.sh [hunk 4]
```diff
@@ -8,4 +8,6 @@ mkdir build || true && \
     cmake -G Ninja .. && \
     cmake --build . && \
     cd .. && \
-    mvn test
+    #mvn  '-Dtest=WriterExtraTablesBenchmarkTest*' test
+    # mvn  '-Dtest=TableTest*' test
+    mvn '-Dtest=WriterTest#canInsertNullRows' test
```
Comment (mkurdej) [inline test.sh]: Euuh, isn't it a change that shouldn't have been committed? It seems as it runs a single test only, nope?

File: src/test/java/net/quasardb/qdb/ts/WriterAppendBenchmarkTest.java (hunk 3 above)
Comment (solatis) [reply src/test/java/net/quasardb/qdb/ts/WriterAppendBenchmarkTest.java]: I apologize, this was temporarily to have some benchmarks inside the unit test suite. It's already been refactored out as a separate component.

File: test.sh (hunk 4 above)
Comment (solatis) [reply test.sh]: You are correct, this script isn't used anywhere in CI, just a convenience script for development. Will undo the change.

File: src/main/c++/net/quasardb/qdb/jni/byte_buffer.cpp (hunk 1 above)
Comment (solatis) [reply src/main/c++/net/quasardb/qdb/jni/byte_buffer.cpp:1]: Did I add you to the reviewers list? I think not.

**Pull request merged**